src/db/models/User.js

```diff
@@ -27,6 +27,7 @@ export default (sequelize, DataTypes) => {
       },
       email: {
         type: DataTypes.STRING,
+        allowNull: true,
```

Comment:
How does this affect the other non-social way of signing up?

Reply:
Those methods are validated, which means that signup operations will not be successful unless an email is provided.